src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorApi.java

```diff
@@ -69,6 +70,13 @@
           "The list of YAML configuration files for the fragment generator.",
           ImmutableList.<String>of());
 
+  public static final Option<String> AUTH_INSTRUCTIONS_URL =
+      ToolOptions.createOption(
+          String.class,
+          "auth_url",
```

Comment:
Wonder whether it would be more correct to call this `auth_instructions`. `auth_url` sounds like a functional URL that you go to to perform a step in the auth process.